tools/repology.json

```diff
@@ -14195,7 +14163,7 @@
     "name": "rpcsvc_proto",
     "description": "rpcsvc protocol definitions from glibc",
     "homepage": "https://github.com/thkukuk/rpcsvc-proto",
-    "version": "1.4.4",
+    "version": "1.4.4-1",
```

Comment:
Does this need to be changed?

Reply:
yes, because we rebuilt the package and incremented the version number